### ISSUE TITLE
Storing PDF and screenshots accept a flag for custom storage path

### ISSUE THREAD

**Issue by joseconstela** [joseconstela/webparsy#25]
Accept the following schema to specify the path of a PDF and screenshot, instead of using a hardcoded value:

```yml
version: 1
jobs:
  main:
    steps:
      - goto: 
          url: http://example.com
      - screenshot:
          path: 
            flag: screenshotLocation
```

**Comment by joseconstela** [comment]
To be released with 1.0

**Comment by joseconstela** [comment]
This fix also made possible to use all options for puppeteer api for [pdf](https://github.com/puppeteer/puppeteer/blob/master/docs/api.md#pagepdfoptions) and [screenshots](https://github.com/puppeteer/puppeteer/blob/master/docs/api.md#pagescreenshotoptions)
